jib-core/examples/README.md

```diff
@@ -4,6 +4,14 @@ Please [file an issue](/../../issues/new) if you find any problems with the exam
 
 For examples on using Jib plugins, see [examples](../../examples).
 
+### [sbt](https://www.scala-sbt.org/) plugin
```

Comment:
We should probably link directly to the source somewhere since it's intended to be an example for jib-core.

Reply:
Switched up the links a bit so that the project name links to the GitHub project.